runtimevar/example_test.go

```diff
@@ -102,6 +103,30 @@ func ExampleSnapshot_As() {
 	}
 }
 
+func ExampleVariable_ErrorAs() {
+	// This example is specific to the runtimeconfigurator implementation; it
+	// demonstrates access to the underlying google.golang.org/grpc/status.Status
+	// type.
+	// The types exposed for As by runtimeconfigurator are documented in
+	// https://godoc.org/gocloud.dev/runtimevar/runtimeconfigurator#hdr-As
+
+	ctx := context.Background()
+
+	const path = "runtimeconfigurator://proj/wrongconfig/key?decoder=string"
```

Comment:
Nit: `path` -> `url`? Or put it inline to avoid naming it. You can also leave off the decoder param, there's a default.

Reply:
Done